pages/live/live.js

```diff
@@ -92,6 +93,38 @@ function setup () {
   })
 
   document.querySelector('#transcript-star').addEventListener("click", handleTranscriptStarred)
+
+  // user actions on auto-summary
+  document.querySelector('#auto-summary-copy-button').addEventListener("click", () => {
+    const summary_text = document.querySelector('#auto-summary-container').textContent.trim()
+    if (summary_text.length) {
+      window.navigator.clipboard.writeText(summary_text)
+        .then(() => toast('copied!', 2000, '#3a5', '#fff'))
+        .catch(() => toast('could not copy summary to clipboard', 3000, '#c00', '#fff'))
+    }
+  })
+  document.querySelector('#auto-summary-inject-button').addEventListener("click", () => {
+    const summary_text = document.querySelector('#auto-summary-container').textContent.trim()
+    if (summary_text.length) {
+      chrome.runtime.sendMessage(_extensionId, {
+        'type': MESSAGE_TYPES.EXTERNAL_TO_BACKGROUND_TEXT_SUMMARY,
+        'summary': summary_text,
```

Comment:
optional, but if you want to get ahead of it at some point in the future there is going to be more finalization stuff (notably dispositions). we have a bit of a choice of if we want to have separate control flow for them or not, and if we want to have that have separate `chrome.runtime.sendMessage` `types`. if we want to jam them all into some sort of finalization payload, the variables should be named correspondigly.

Reply:
Does https://github.com/anene/trellus/pull/100 look like what we want for that?